tasks/validate-single-component/README.md

```diff
@@ -0,0 +1,10 @@
+# validate-single-component
+
+Tekton task validates that the snapshot only contains a 
```

Comment:
Can you make this begin`A tekton task that validates` to make it a better sentence?

Reply:
Sure, Done!